shared/common-adapters/popup-dialog.desktop.js

```diff
@@ -10,13 +10,13 @@ function stopBubbling (ev) {
   ev.stopPropagation()
 }
 
-export function PopupDialog ({children, onClose, fill}: Props) {
+export function PopupDialog ({children, onClose, fill, styleCover, styleContainer, styleClose, styleClipContainer, allowClipBubbling}: Props) {
   return (
     <EscapeHandler onESC={onClose}>
-      <Box style={styleCover} onClick={onClose}>
-        <Box style={{...styleContainer, ...(fill ? styleContainerFill : null)}}>
-          <Icon type='iconfont-close' style={styleClose} />
-          <Box style={styleClipContainer} onClick={stopBubbling}>
+      <Box style={{...coverStyle, ...styleCover}} onClick={onClose}>
+        <Box style={{...containerStyle, ...(fill ? containerFillStyle : null), ...styleContainer}}>
+          <Icon type='iconfont-close' style={{...closeStyle, ...styleClose}} />
+          <Box style={{...clipContainerStyle, ...styleClipContainer}} onClick={allowClipBubbling ? undefined : stopBubbling}>
```

Comment:
the intention is the cover gets the click so it closes (right now the cover's onClick never fires). This is mostly an artifact of us treating the container style differently. I'll fix that up and leave this i think)